README.md

```diff
@@ -121,6 +121,86 @@ steps:
           pull-request-metadata-key: "github-pull-request-plugin-number"
 ```
 
+## Example Pipeline
+
+This is an example pipeline which ties everything together to produce nicely
+annotated bundle update pull requests.
+
+This pipeline requires two secrets:
+
+* Write access to the project GIT repository, by way of an SSH Key. This write
+  access is used for pushing up the bundle update commit.
```

Comment:
Can we link to the [deploy key documentation](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) to give people a hint on how to create one?

Reply:
Yes. Good idea.

Reply:
Added in b7c0625.